tuftool/src/main.rs

```diff
@@ -39,6 +44,8 @@ enum Command {
     Refresh(refresh::RefreshArgs),
     /// Download a TUF repository's resources
     Download(download::DownloadArgs),
+    /// Update a TUF repository's metadata and targets
```

Comment:
```suggestion
    /// Update a TUF repository's metadata and optionally add targets
```
Feels like it's good to clarify that adding targets is the thing they can do; don't want someone to think it's a full replacement of targets.